README.md

```diff
@@ -330,18 +330,37 @@ Once it's ready user can define
 `--type` - defines a type of a storage, currently xpk supports `gcsfuse` only.
 `--auto-mount` - if set to true means that all workloads should have a given storage mounted by default.
 `--mount-point` - defines the path on which a given storage should be mounted for a workload.
-`--manifest` -- 
+`--manifest` - defines the path to manifest which contains PersistentVolue and PersistentVolumeClaim definitions
 
 
-* Create simple Storage
+* Create a simple Storage
 
     ```shell
     python3 xpk.py storage create test-storage --project=$PROJECT
-    --cluster=xpk-test --type=test-type --auto-mount=false \
+    --cluster=xpk-test --type=gcs-fuse --auto-mount=false \
```

Comment:
I think that instead of gcs-fuse this should be gcsfuse.

Reply:
Done